src/controllers/user/signin.ts

```diff
@@ -0,0 +1,19 @@
+import type { Request, Response } from 'express';
+import User from '../../database/user';
+
+const signIn = async (req:Request, res:Response): Promise<void> => {
+  try {
+    const { username, password } = req.body;
+    const user = await User.findOne({ username });
+    if (user?.password === password) {
+      res.status(200).json({
+        user,
+        msg: 'logged',
+      });
+    }
+  } catch (error) {
+    console.log(error);

```

Comment:
custom error not log message

Reply:
sign in is not ready yet, all sign in code need to be changed, because sign up was not ready and there is no jwt sign